src/deluge/hid/led/pad_leds.h

```diff
@@ -113,15 +114,15 @@ void setupAudioClipCollapseOrExplodeAnimation(AudioClip* clip);
 
 void setGreyoutAmount(float newAmount);
 
-static inline void flashMainPad(int32_t x, int32_t y, int32_t color = 0) {
-	if (color > 0) {
-		bufferPICUart(10 + color);
+static inline void flashMainPad(int32_t x, int32_t y, int32_t colour = 0) {
```

Comment:
perhaps this could be moved to the new PIC namespace instead? then the intermediate new `flashPad(idx)` won't be needed. (IMO MainPad is a better name as to show this is a hardware limitation)

Reply:
I think it becomes an issue of scope: PadLEDs provides the high-level interface (coordinate, optional color), while the PIC interface merely sends the message correctly.